frontend/src/queries/schema/schema-general.ts

```diff
@@ -3112,6 +3124,7 @@ export type InsightFilter =
     | StickinessFilter
     | LifecycleFilter
     | CalendarHeatmapFilter
+    | WebAnalyticsPropertyFilters
```

Comment:
Same as above - the filter type is there, since all insights have a filter key that contains the more advanced configuration. E.g. `trendsFilter` for trend insights. Web analytics queries are built on a different structure and thus should not have an insight filter.